README.md

```diff
@@ -83,7 +83,8 @@ After one month training.
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-[Live](https://osmanwako.github.io/Erriechafestival/)
+[Live](https://osmanwako.github.io/Project-Erriecha/)
+<!-- project loom video https://www.loom.com/share/c3e272ceea414a4a812fbd7269f02bb2 -->
 
```

Comment:
- [ ] Kindly add video documentation of your project as stated in the project requirement. Keep the following points in mind when recording the video. 🤓️

![image](https://user-images.githubusercontent.com/17243325/227477609-77ed08e2-c3ca-4c01-bbc3-c4d10717e0aa.png)

Reply:
- [ ] @osmanwako great work on the presentation, however, the presentation is more than the recommended time of 5 minutes as shown below
![image](https://user-images.githubusercontent.com/61408860/227577127-1c6d078d-66c1-4a10-a38e-a9867218f3ed.png)

-kindly ensure your presentation follows all the guidelines below as previously requested
![image](https://user-images.githubusercontent.com/61408860/227577446-0ddb310d-0f8d-4d98-9574-9b658c9eb6e8.png)